packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -826,6 +867,11 @@ export class TerminalFrontendContribution implements FrontendApplicationContribu
             command: TerminalCommands.TOGGLE_TERMINAL.id,
             keybinding: 'ctrl+`',
         });
+        keybindings.registerKeybinding({
+            command: TerminalCommands.SELECT_ALL.id,
+            keybinding: 'ctrlcmd+a',
+            context: TerminalKeybindingContexts.terminalActive
```

Comment:
minor: `context` is deprecated, we should probably make use of `when` clauses instead.

Reply:
Right, I didn't use the `when` to maintain consistency, but you're right, better to start at some point. As far as I could tell, the context key `terminalFocus` is equivalent, so I took the liberty to replace all usages of `context: TerminalKeybindingContexts.terminalActive` in this class.